openedx/features/badging/models.py

```diff
@@ -53,54 +49,81 @@ def get_badges_json(cls, badge_type):
         return JSONRenderer().render(badges)
 
 
+class UserBadgeManager(models.Manager):
+
+    def create_user_badge(self, user_id, badge_id, course_id, community_id):
+        """Create user badge on badge assignment only if user badge does not exists."""
+        return self.model.objects.get_or_create(
```

Comment:
this method and the default manager's get_or_create has only args vs kwargs difference. in my opinion that does not constitute as different enough to write a new manager